tests/Tests/600__ai_apps/670__pachyderm/670__pachyderm.robot

```diff
@@ -15,37 +14,99 @@ ${pachyderm_appname}            pachyderm
 
 
 *** Test Cases ***
-Verify Pachyderm Can Be Installed Using OpenShift Console
-    [Documentation]     Check if it is possible to install and deploy pachyderm server successfully.
+Verify Pachyderm Can Be Installed And Deploy Its Server
+    [Documentation]     Check if it is possible to install pachyderm operator and it's server can be deployed.
     [Tags]      Tier2
     ...         ODS-1137    ODS-1138
-    Check And Install Operator in Openshift    ${pachyderm_container_name}    ${pachyderm_appname}
-    Create Project      pachyderm
-    Create Pachyderm AWS-Secret
-    Create Tabname Instance For Installed Operator        ${pachyderm_container_name}   ${pachyderm_container_name}     pachyderm
-    Wait Until Status Is Running
+    Pass Execution      Passing test, as suite setup ensures Pachyderm operator is installed correctly.
+
+Verify Pachyderm Pipeline Can Be Created
+    [Documentation]     Check if it is possible to create smaple pipline using jupyterlab.
+    [Tags]      Tier2
+    ...         ODS-1161
+    Go To    ${OCP_CONSOLE_URL}
+    ${version}=     Get Pachd Version
     Go To RHODS Dashboard
     Verify Service Is Enabled    Pachyderm
-    [Teardown]  Uninstall Pachyderm Operator
+    Launch JupyterHub Spawner From Dashboard
+    Spawn Notebook With Arguments  image=s2i-generic-data-science-notebook
+    Create Pachyderm Pipeline Using JupyterLab     ${version}
+    Verify Pipline Pod Creation
+    [Teardown]  Delete Pipeline And Stop JupyterLab Server
 
 *** Keywords ***
 Dashboard Test Setup
     Set Library Search Order    SeleniumLibrary
     Open Browser    ${OCP_CONSOLE_URL}    browser=${BROWSER.NAME}    options=${BROWSER.OPTIONS}
     Login to OCP
     Wait Until OpenShift Console Is Loaded
+    Check And Install Operator in Openshift    ${pachyderm_container_name}    ${pachyderm_appname}
+    Create Project      pachyderm
+    Create Pachyderm AWS-Secret
+    Create Tabname Instance For Installed Operator        ${pachyderm_container_name}   ${pachyderm_container_name}     ${pachyderm_appname}
+    Wait Until Status Is Running
 
 Dashboard Test Teardown
+    Go To    ${OCP_CONSOLE_URL}
+    Delete Tabname Instance For Installed Operator    ${pachyderm_operator_name}    ${pachyderm_operator_name}      ${pachyderm_appname}
+    Uninstall Operator    ${pachyderm_operator_name}
+    Delete Project By Name      pachyderm
+    Launch Dashboard    ocp_user_name=${TEST_USER.USERNAME}    ocp_user_pw=${TEST_USER.PASSWORD}
+    ...    ocp_user_auth_type=${TEST_USER.AUTH_TYPE}    dashboard_url=${ODH_DASHBOARD_URL}    browser=${BROWSER.NAME}
+    ...    browser_options=${BROWSER.OPTIONS}
+    Remove Disabled Application From Enabled Page    app_id=pachyderm
```

Comment:
same logic as Setup: the pachyderm uninstall should be in the Suite Setup rather than Test Setup, otherwise the it would be triggered twice

Could you also pls change the kw title?

Reply:
Done !